Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -226,6 +238,23 @@ private Frame GetFrame()
             return _frame ?? (_frame = this.FindVisualAscendant<Frame>());
         }
 
+        private void MasterDetailsViewViewStateChanged(object sender, MasterDetailsViewState e)
+        {
+            if (e == MasterDetailsViewState.Details)
+            {
+                // If the master list or any of its descendents have focus, then force the focus to the next item and
+                // ensure it has moved outside of the master list. Otherwise, keyboard movement with still occur on
+                // the master list even though we're in details view
+                while (IsFocusInControl(_masterList))
+                {
+                    if (!FocusManager.TryMoveFocus(FocusNavigationDirection.Next))
```

Comment:
This works, but I would rather force the focus onto details view if it was focused on the master list. However, I can't figure out how to apply focus to `_detailsPresenter` which is a `ContentPresenter`